src/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -862,5 +864,11 @@ private static Complex Scale(Complex value, double factor)
         {
             return new Complex((double)value, 0.0);
         }
+
+        public static bool IsFinite(Complex value) => !Infinity.Equals(value);
+
+        public static bool IsNaN(Complex value) => NaN.Equals(value);
+
+        public static bool IsInfinity(Complex value) => Infinity.Equals(value);
```

Comment:
A complex or imaginary value with at least one infinite part is regarded as an infinity (even if its other part is a NaN).

Reply:
This can probably be implemented using `=> double.IsInfinity(value.m_real) || double.IsInfinity(value.m_imaginary)`

Reply:
Thanks. I went with the narrowly defined value of `Infinite` being `(+Inf, +Inf)` per the issue discussion because I didn't want to overstep. Figured I would raise the question of whether this is what we are really after in the PR description. Change made and PR description revised.